### PR TITLE
Release 15.2.1

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [15.2.1](https://github.com/theforeman/puppet-foreman_proxy/tree/15.2.1) (2020-10-15)
+
+[Full Changelog](https://github.com/theforeman/puppet-foreman_proxy/compare/15.2.0...15.2.1)
+
+**Fixed bugs:**
+
+- Do not set ACLs on DHCP leases directory [\#627](https://github.com/theforeman/puppet-foreman_proxy/pull/627) ([lzap](https://github.com/ekohl))
```
Comment (ekohl) [inline CHANGELOG.md:9]: I'm hesitant to classify this as a bugfix, which is why I suggested a minor release (15.3.0). Also, it wasn't written by lzap. (same below)

**Review by ekohl — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ehelms) [reply CHANGELOG.md]: The joys of creating changelogs by hand. The commit message for that change reads like it fixes a bug.

Comment (ekohl) [reply CHANGELOG.md]: Setting the ACLs is redundant, but I wouldn't classify it as a bugfix.

**Comment by ehelms** [comment]
I am interpreting the comments here as requesting a 15.3.0 despite https://github.com/theforeman/puppet-foreman_proxy/pull/627 having been approved.

**Pull request closed without merging**